cypress/support/pages/tasks/pageTasks.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+class Task {
```

Comment:
may be here class name can be in plural too like in file name --> Tasks {}

Reply:
I changed it to Tasks